src/client/pages/OfferNew/Introduction/Sidebar/index.tsx

```diff
@@ -231,7 +230,7 @@ export const Sidebar: React.FC<Props> = ({
                 />
               </Header>
               <Body>
-                {hasMultipleQuotes && <PriceBreakdown offerData={offerData} />}
+                {<PriceBreakdown offerData={offerData} />}
```

Comment:
I think we can remove the curly brackets now as it should always be displayed :)
```suggestion
                <PriceBreakdown offerData={offerData} />
```